### DISCUSSION TITLE
Can we install zepp os apps on mi band 8?

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#143] [General]
If mi band 7 can install apps so why mi band 8 can't?

I watched the live stream of Xiaomi in which they were introducing mi band 8. They also mentioned that it supports installing games .

**Comment by mimimix** [comment]
...and what version of zepp os is there?

**Comment by MyoungXUE** [comment]
Hello, mi band 8 is not equipped with Zepp OS system, so you can't install Zepp OS application.
